charts/datadoc/values.schema.json

```diff
@@ -175,6 +179,7 @@
           "description": "Password",
           "default": "changeme",
           "render": "password",
+          "hidden": true,
```

Comment:
I would prefer if we just remove the prop(.Values.security.password) and any reference to it. Can't see a use for it anymore.